themes/theme-b2c-tailwind/src/molecules/card.css

```diff
@@ -0,0 +1,22 @@
+[data-store-card] {
```

Comment:
Are we going to use sass inside a tailwind project? I feel a little confused about that. Should we try to create another repo for sass theme end setting up on storybook as a theme as well?

Reply:
For now, no. Maybe in the future.

I will just copy what we [discussed about it](https://github.com/vtex/faststore/pull/1272#issuecomment-1124316618).

> Nowadays as the styles are only used in the documentation components, it doesn't seem to be a problem to have this gradual transition within the tailwind project itself. Gradually, we are migrating the styles without having to create a new theme and configure it in Storybook.